test/services/teams/events/index.test.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable new-cap */
+
+const { expect } = require('chai');
+const { ObjectId } = require('mongoose').Types;
+const sleep = require('util').promisify(setTimeout);
+
+const app = require('../../../../src/app');
+const { userModel } = require('../../../../src/services/user/model');
+const { teamsModel, teamUserModel } = require('../../../../src/services/teams/model');
+
+const DELAY_TIME = 250;
+const schoolId = ObjectId();
+const userRoleId = ObjectId();
+const teamRoleId = ObjectId();
+
+const createTeamUser = userId => (new teamUserModel({ role: teamRoleId, schoolId, userId }))._doc;
+
+const createUser = () => {
+	const time = Date.now();
+	return userModel.create({
+		firstName: time,
+		lastName: 'test',
+		email: `${time}.test@test.de`,
+		roles: [userRoleId],
+		schoolId,
+	}).then(user => user._doc);
+};
+
+const createTeam = owner => teamsModel.create({
```

Comment:
Eventuell in `testObjects` einpflegen (sieht ja recht übersichtlich aus).

Reply:
Kann ich machen, weitere Meinungen dazu? Wo werden Teams noch für Test verwendet? In news?
@schurigh @Metauriel

Reply:
Stimme Max zu, die testObjekte können wir alle in einem Modul zusammenbringen.
Das werde ich wohl auch in rocketchat tests brauchen...

Reply:
Ich finde den Lösungsansatz gut, aber würde gerne erst den helper im Architektur Meeting noch mal durchsprechen. Daher eine Lösung über den helper in einem späteren PR einbauen wollen.